src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -128,10 +145,46 @@ class TableauxModel(override protected[this] val connection: DatabaseConnection)
 
       _ <- updateRowModel.clearRow(table, rowId, Seq(column))
 
+      _ <- invalidateCellAndDependentColumns(column, rowId)
+
       replacedCell <- retrieveCell(column, rowId)
     } yield replacedCell
   }
 
+  val CACHING = true
```

Comment:
Will be removed in near future I guess. I think I will add an layer between `TableauxModel` and `RowModel`.